README.md

```diff
@@ -558,6 +558,8 @@ const uiSchema = {
 };
 ```
 
+In addition it is valid to mark an array as "required".  This means the user must select at least one item to add to the array.
```

Comment:
I am in favor of this change. It is consequent with the empty string to undefined functionality. The error message you get when you don't select a value is then always the same, whether you never selected a value or you selected and then deselected a value. If something looks the same in the ui, I think it should be the same.

For JSON Schema, the empty array is of course a valid array, but JSON Schema is not a form builder and the difference between an empty array and undefined is simply not visible in a form and should as a consequence not be settable through the form, in my opinion.